src/main/java/Deadline.java

```diff
@@ -0,0 +1,33 @@
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+
+public class Deadline extends Task {
+    protected String by;
+    protected LocalDate dueDate;
+    protected LocalTime timing;
```

Comment:
Good use of separate variables for date and time. Possible future implementation to make timing optional?